src/Tasks.UnitTests/ResolveComReference_Tests.cs

```diff
@@ -753,7 +753,11 @@ public void AxReferenceKnowsItsRCWPiaExists_IncludeVersion()
             CheckAxReferenceRCWTlbExists(RcwStyle.PreexistingPia /* pass in the PIA reference */, true /* include version in the interop name */);
         }
 
-        private enum RcwStyle { GenerateTlb, PreexistingTlb, PreexistingPia };
+        private enum RcwStyle {
+            GenerateTlb,
+            PreexistingTlb,
+            PreexistingPia
+        };
```

Comment:
📝 All the enums in this pull request should use the consistent style for C# projects (Allman-style braces, no extraneous semicolon). I would encourage, but not require, the use of a trailing comma on the final enum item. Here's the result of all three suggestions applied to this case:

```suggestion
        private enum RcwStyle
        {
            GenerateTlb,
            PreexistingTlb,
            PreexistingPia,
        }
```

Reply:
> 

Done.